<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockDialog</class>
 <widget class="QDialog" name="BlockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Block</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="titleEdit">
     <property name="placeholderText">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="startTimeEdit"/>
   </item>
   <item>
    <widget class="QTimeEdit" name="endTimeEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="colorButton">
     <property name="text">
      <string>Select Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionEdit">
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
